system-tests/projects/todos/cypress/support/component-index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```

Comment:
I noticed a bunch of compilation errors around the failing spec due to this file missing from the todos project. `todos` is configured for component testing but did not include this file. I don't think this was causing a problem in the tests outside of the chatty log, but now it's set up correctly and the log is chatty no more.